### ISSUE TITLE
Uncaught TypeError: Cannot read property 'querySelector' of null

### ISSUE THREAD

**Issue by MrJadaml** [thomaslindstrom/color-picker#56]
Uncaught TypeError: Cannot read property 'querySelector' of null

**Atom Version**: 0.156.0
**System**: Mac OS X 10.9.5
**Thrown From**: [color-picker](https://github.com/thomaslindstrom/color-picker) package, v1.4.4

### Steps To Reproduce

1. ...
2. ...

### Stack Trace

```
At /Users/Jadam/.atom/packages/color-picker/lib/ColorPicker-view.coffee:101

TypeError: Cannot read property 'querySelector' of null
  at ColorPickerView.module.exports.ColorPickerView.open (/Users/Jadam/.atom/packages/color-picker/lib/ColorPicker-view.coffee:101:66)
  at Object.module.exports.open (/Users/Jadam/.atom/packages/color-picker/lib/ColorPicker.coffee:108:23)
  at atom-text-editor.atom.commands.add.color-picker:open (/Users/Jadam/.atom/packages/color-picker/lib/ColorPicker.coffee:20:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:243:29)
  at /Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:549:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:391:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:167:20)

```

/cc @atom/core

**Comment by thomaslindstrom** [comment]
I think this is pretty much fixed when you update Atom to the current 0.175.0 version. `color-picker` v 1.4.4 converted to the new Atom 1.0.0 APIs, which may be causing the issue.

Please reopen if you still have problems after updating.

**Comment by thiago-om** [comment]
I am still having this issue:

```
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.176.0
**System**: Mac OS X 10.10.2
**Thrown From**: [color-picker](https://github.com/thomaslindstrom/color-picker) package, v1.4.4

### Stack Trace

Uncaught TypeError: Cannot read property 'querySelector' of null

```
At /Users/Thiago/.atom/packages/color-picker/lib/ColorPicker-view.coffee:101

TypeError: Cannot read property 'querySelector' of null
  at ColorPickerView.module.exports.ColorPickerView.open (/Users/Thiago/.atom/packages/color-picker/lib/ColorPicker-view.coffee:101:66)
  at Object.module.exports.open (/Users/Thiago/.atom/packages/color-picker/lib/ColorPicker.coffee:108:23)
  at atom-text-editor.atom.commands.add.color-picker:open (/Users/Thiago/.atom/packages/color-picker/lib/ColorPicker.coffee:20:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:243:29)
  at /Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:558:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:396:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:170:20)

```

### Commands

```
     -0:08.1 core:backspace (input.hidden-input)
     -0:07.9 core:move-up (input.hidden-input)
     -0:07.5 editor:move-to-end-of-screen-line (input.hidden-input)
     -0:07.2 core:backspace (input.hidden-input)
     -0:07.0 core:move-up (input.hidden-input)
     -0:06.4 editor:move-to-end-of-screen-line (input.hidden-input)
     -0:05.8 core:backspace (input.hidden-input)
     -0:05.7 core:move-up (input.hidden-input)
     -0:05.2 editor:delete-to-beginning-of-line (input.hidden-input)
     -0:04.6 core:undo (input.hidden-input)
     -0:04.1 core:backspace (input.hidden-input)
     -0:03.7 core:save (input.hidden-input)
  8x -0:03.2 core:move-down (input.hidden-input)
     -0:02.4 emmet:insert-formatted-line-break-only (input.hidden-input)
     -0:02.4 editor:newline (input.hidden-input)
     -0:00.9 color-picker:open (input.hidden-input)
```

### Config

```json
{
  "core": {
    "ignoredNames": [
      "node_modules",
      ".idea",
      ".git",
      ".DS_Store",
      "Thumbs.db"
    ],
    "followSymlinks": true,
    "disabledPackages": [
      "atom-beautify",
      "atom-jade",
      "emmet"
    ],
    "themes": [
      "one-light-ui",
      "base16-tomorrow-light-theme"
    ],
    "projectHome": "/Users/Thiago/Sites/thiago-om/src"
  }
}
```

### Installed Packages

```coffee
# User
Atom-Syntax-highlighting-for-Sass, v0.5.0
Rdio, v0.5.1
atom-handlebars, v0.4.0
auto-update-packages, v0.2.2
autocomplete-paths, v1.0.1
autocomplete-plus, v2.0.6
autocomplete-snippets, v1.0.0
color-picker, v1.4.4
dash, v1.0.2
editor-stats, v0.16.0
emmet-atom, v2.3.2
gist-it, v0.6.4
git-log, v0.2.0
gulp-snippets, v0.6.0
javascript-snippets, v1.0.0
language-jade, v0.3.0
linter, v0.10.1
minimap, v3.5.6
minimap-git-diff, v3.1.1
one-light-ui, v0.0.0
pretty-json, v0.3.2
project-manager, v1.15.5
remote-edit, v1.6.3
sassbeautify, v0.2.0
tabs-to-spaces, v0.8.1
terminal-status, v1.3.5
wordpress-api, v1.1.0

# Dev
No dev packages
```

/cc @atom/core
```
